robot_manager:
	ros__parameters:
		joint_stiffness: [1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0]
		
